k8_cortx_cloud/logs-cortx-cloud.sh

```diff
@@ -115,10 +115,20 @@ function tarPodLogs()
     local container=$1
 
     printf "\n ⭐ Generating support-bundle logs for pod: %s\n" "${pod}"
+    # shellcheck disable=SC2016
     kubectl exec "${pod}" -c "${container}" --namespace="${namespace}" -- \
-      cortx_support_bundle generate \
-        --cluster_conf_path yaml:///etc/cortx/cluster.conf \
-        --location file://${path} \
+      env location="file://${path}" \
```

Comment:
I don't really like this, so open to suggestions.

Reply:
I don't have any better ideas.  We'll probably want to take another look at logs-cortx-cloud.sh in the future anyway, so we might have an opportunity to change (or just remove the script).

Reply:
If this works, I'm good with it for now (to avoid overthinking it). We'll want to reassess logs and support bundle as a whole anyway, as Walter mentions, so we can optimize it then.